Fix missing latlng on mouse event in latest leaflet version

Pierstoval opened the pull request:
The project doesn't seem properly maintained for now, but I'm using it and this PR allows fixing bugs in click events when using latest leaflet versions and canvas renderer.

Here for sake of people needing it, just in case.

Pierstoval committed:
Fix missing latlng on mouse event in latest leaflet version

IvanSanchez commented:
Closing for the same reason as https://github.com/Leaflet/Leaflet/pull/8888 - inability to provide a reproducible minimal example.

Pierstoval commented:
As stated in Leaflet/Leaflet#8888, we rewrote everything because VectorGrid was erroneous in terms of interactivity and we couldn't find anything in the lib to make it work 100% (styling was also erronenous in some cases).

Switched to GeoJSON + events that call GeoServer on each map pan/zoom, and it's fine for now.

Thanks anyway for the review, at least we now know that the problem might have been more a PEBKAC 😅
